chart/values.yaml

```diff
@@ -395,8 +395,6 @@ pgbouncer:
   logConnections: 0
 
 redis:
-  # set to true if redis resources/service should be created
-  enabled: false
   terminationGracePeriodSeconds: 600
```

Comment:
for the `enabled`, I followed the same syntax that there currently exists for db (though I think it's not really logical indeed since to sense to give url + enabled = true), it might be more aligned with their conventions

Reply:
I would completly refactor the values personnaly. it's a bit of a mess ; but that's for another time :D (redis is enabled when you have celery executor selected).
Let's discuss this tomorrox